scripts/main.js

```diff
@@ -19,7 +19,7 @@
         if (ev.data.jacks_message === null) {
```

Comment:
found a better fix, this === should have been only ==

Reply:
Probably the better expression then is `if (!ev.jacks_message)` if you want to indicate that **any** falsy value should enter this branch.